src/pages/ReportView/WorkloadSummary/WorkloadSummary.tsx

```diff
@@ -208,7 +208,7 @@ class WorkloadMigrationSummary extends React.Component<Props, State> {
                     />
                     <SolidCard
                         title={`${formatPercentage(percentages[3], 0)} OCP`}
-                        description="Workloads suitable for Container-Native Virtualization"
+                        description="Workloads suitable for OpenShift virtualization"
```

Comment:
Sorry @carlosthe19916, i think there has been a little bit of misunderstanding about the final label.
```suggestion
                        description="Workloads targeted for OpenShift virtualization"
```